plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectConfig.java

```diff
@@ -114,4 +115,12 @@ String getMavenArtifactId() {
   String getMavenVersion() {
     return mavenVersion;
   }
+
+  void setEntryPoint(String entryPoint) {
```

Comment:
I didn't think you need to set a custom entrypoint. The [quickstart guide](https://cloud.google.com/appengine/docs/standard/java11/quickstart) does say anything about it, and the sample there doesn't have anything related to an entrypoint. Is this needed? Perhaps I don't understand the App Engine Java 11 runtime.

And will we expose this field in the wizard UI?

Reply:
It's needed for thin jars. I think we'll eventually expose it in the UI. See https://cloud.google.com/appengine/docs/standard/java11/runtime